src/platform/Linux/ConnectivityManagerImpl.cpp

```diff
@@ -1610,10 +1612,23 @@ CHIP_ERROR ConnectivityManagerImpl::_WiFiPAFSubscribe(const uint16_t & connDiscr
             return self->OnNanSubscribeTerminated(term_subscribe_id, reason);
         }),
         this);
+    static constexpr System::Clock::Timeout kNewConnectionScanTimeout = System::Clock::Seconds16(20);
```

Comment:
This is a shorter timer than the actual discovery timeout in SetUpCodePairer.

Why are we doing this?  Why are we not just allowing the existing timer in SetUpCodePairer to do its work?

Reply:
It's to reference the definition in BLEManageImpl.cpp:
static constexpr System::Clock::Timeout kNewConnectionScanTimeout = System::Clock::Seconds16(20);

This is required if using wifipaf-wifi, instead of code-wifi

Reply:
As I said above: the timeout needs to be an upper-layer policy decision.  If `wifipaf-wifi` (which to be clear: maybe shouldn't even exist) needs this, then _it_ should be setting a timer as desired.  Probably one with chip-tool command-line control over it.